packages/cli/test/analytics.test.ts

```diff
@@ -1,12 +1,19 @@
-import Login from '@heroku-cli/plugin-auth/src/commands/auth/login'
-import * as Config from '@oclif/config'
+import {Config} from '@oclif/core'
 import {expect} from 'chai'
 import nock from 'nock'
 import sinon from 'sinon'
 
 import AnalyticsCommand, {AnalyticsInterface} from '../src/analytics'
 import UserConfig from '../src/user-config'
 
+const mockCommand = {
+  plugin: {
+    name: 'foo',
+    version: '123',
+  },
+  id: 'login',
+}
+
```

Comment:
I want to explain this addition and the changes to the file below a bit. There was a Typescript error when using `Login` for the `Command` item in the parameter object for the `analytics.record` function. I tried everything I could think of to fix it, but nothing was working. Looking at the now-deleted code on line 36, however, I got the idea to add `as any` to `Login`, so that the Command item would look like this: `Command: Login as any`. This worked! The tests passed, and I tested the analytics functionality manually and it was working perfectly fine.

I also did not really care for importing the Login function just for testing. And if we were going to have to add an `as any` anyway, I thought I might as well just go ahead and get rid of that import and use a `mockCommand` object instead.

I am open to other solutions here, but this is the best option I could come up with.

Reply:
Nice!